.github/workflows/tests.yml

```diff
@@ -5,7 +5,7 @@ on: [pull_request]
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     strategy:
       matrix:
         python-version: [3.8]
```

Comment:
This project looks to be quite out of date - I think 3.8 has been EOL for awhile, we probably need to update all the dependencies.

Reply:
Yeah. I was thinking similar things. There's also #19 reported recently, where installation is failing due to some deprecations. Increasing the Python version feels like a thing to do as part of addressing that issue.

Probably needs some dedicated attention, though. Should I write a ticket for the Harmony backlog?